csaf-validation/src/main/kotlin/io/github/csaf/sbom/validation/tests/Extensions.kt

```diff
@@ -145,6 +162,18 @@ internal operator fun <E> Collection<E>?.plus(other: Collection<E>?): Collection
     } else if (this != null) {
         this
     } else {
-        listOf()
+        setOf()
```

Comment:
Are we sure that we never expect this operator to return a list (or a collection which can have duplicate elements)? There are various tests checking for duplicate entries...

Reply:
Well, this is basically a wrapper around the regular "plus" that allows null values, the regular plus maps to `union` and `union` always returns a set.